hw/ip/alert_handler/data/alert_handler.hjson.tpl

```diff
@@ -344,7 +344,11 @@ chars = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H']
           ''',
           resval: 1,
         }
-      ]
+      ],
+      tags: [// the value of this register is set to false only by hardware, under the condition
+            // that escalation is triggered and the corresponding lock bit is true
+            // could not be auto-predicted so it is excluded from read check
+            "excl:CsrNonInitTests:CsrExclWriteCheck"]
```

Comment:
nit: spacing off by 1

Reply:
nit: Also use the sentence case on all comments.

Reply:
Oops, thanks for pointing that out. I will update them.